src/js/map/boundary_type_box.js

```diff
@@ -29,22 +42,45 @@ export class BoundaryTypeBox extends Component {
         }
     }
 
-    populateBoundaryOptions = (children, currentLevel) => {
+    populateBoundaryOptions = (children, currentLevel, versions) => {
         if (typeof this.preferredChildren === 'undefined') {
             return;
         }
+        let existingVersions = [...versions].filter((v) => {
+            return v.exists
+        });
 
         let boundaryTypes = Object.keys(children);
+        let options = this.getOptions(boundaryTypes, existingVersions);
 
-        if (typeof this.preferredChildren[currentLevel] !== 'undefined') {
-            const availableLevels = this.preferredChildren[currentLevel].filter(level => children[level] != undefined)
+        if (typeof this.preferredChildren[currentLevel] !== 'undefined' || (boundaryTypes.length === 0 && existingVersions.length > 0)) {
+            //(boundaryTypes.length === 0 && existingVersions.length > 0) -> there are no children but there are multiple versions
+            const availableLevels = this.preferredChildren[currentLevel] !== undefined ? this.preferredChildren[currentLevel].filter(level => children[level] !== undefined) : [];
+            const selectedOption = availableLevels.length > 0 ? `${this.activeVersion.model.name} / ${availableLevels[0]}` : this.activeVersion.model.name;
```

Comment:
There's a lot happening here in a few lines, maybe we can improve readability by reworking the nested if statements or separating them into functions